build.sh

```diff
@@ -183,29 +183,29 @@ mkdir /var
 mkdir /frzr_root
 EOF
 
-echo "${CHANNEL}-${VERSION}" > ${BUILD_PATH}/build_info
-echo "" >> ${BUILD_PATH}/build_info
-cat ${BUILD_PATH}/manifest >> ${BUILD_PATH}/build_info
-rm ${BUILD_PATH}/manifest
+echo "${CHANNEL}-${VERSION}" > "${BUILD_PATH}"/build_info
+echo "" >> "${BUILD_PATH}"/build_info
+cat "${BUILD_PATH}"/manifest >> "${BUILD_PATH}"/build_info
+rm "${BUILD_PATH}"/manifest
 
-btrfs subvolume snapshot -r ${BUILD_PATH} ${SNAP_PATH}
-btrfs send -f ${CHANNEL}-${VERSION}.img ${SNAP_PATH}
+btrfs subvolume snapshot -r "${BUILD_PATH}" "${SNAP_PATH}"
+btrfs send -f "${CHANNEL}"-"${VERSION}".img "${SNAP_PATH}"
```

Comment:
In this case I think it would look better to write `"${CHANNEL}-${VERSION}.img"`

Reply:
yeah is little same just have secure var.